lms/djangoapps/discussion/tasks.py

```diff
@@ -74,6 +75,12 @@ def __init__(self, *args, **kwargs):
         self.options['transactional'] = True
 
 
+class CommentNotification(BaseMessageType):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.options['transactional'] = True
```

Comment:
Transactioanl should be used for important emails like password reset and others. I think in this case, `CommentNotification` isn't transactional

```suggestion
class CommentNotification(BaseMessageType):
    pass
```

Reply:
Fixed, thanks.